documentation/requirements.txt

```
@@ -1,4 +1,4 @@
-sphinx>=4.0.0
+sphinx>=4.0.3
```

```suggestion
sphinx==4.0.3
```
You shoudl have copied it correctly.

---

oh shit